src/main/java/com/example/helpfy/controllers/CommentController.java

```diff
@@ -0,0 +1,128 @@
+package com.example.helpfy.controllers;
+
+import com.example.helpfy.dtos.comment.CommentMapper;
+import com.example.helpfy.dtos.comment.CommentRequest;
+import com.example.helpfy.dtos.comment.CommentResponse;
+import com.example.helpfy.models.Answer;
+import com.example.helpfy.models.Comment;
+import com.example.helpfy.models.Question;
+import com.example.helpfy.services.answer.AnswerService;
+import com.example.helpfy.services.comment.CommentService;
+import com.example.helpfy.services.question.QuestionService;
+import com.example.helpfy.services.user.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/comments")
+@CrossOrigin
+public class CommentController {
```

Comment:
Apenas uma sugestão, mas tem alguns lugares que usar o `this` para referenciar os atributos e outros não, talvez seja interessante padronizar.